### PR TITLE
Migrate from aioredis to redis.asyncio in (demos/shortify, demos/blog)

### PR DESCRIPTION
**Pull request by Magomedov-Magomed**
## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: demos/shortify/setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def read_version():
     'trafaret',
     'aiohttp_jinja2',
     'pyyaml',
-    'aioredis==1.3.1'
```
Comment (Dreamsorcerer) [inline demos/shortify/setup.py:25]: Could you replace this with `redis` as `aioredis` is no longer supported?
Related PRs, which might give some help:
https://github.com/aio-libs/aiohttp-session/pull/761
https://github.com/aio-libs/aiohttp-session/pull/701

Comment (Magomedov-Magomed) [reply demos/shortify/setup.py:25]: Sure. Thanks for the advice and links

**Comment by Dreamsorcerer** [comment]
I'm going to push a new release for aiohttp-session at the weekend to switch to redis there. Then we should be able to get the tests passing here.

**Comment by Dreamsorcerer** [comment]
Alright, if you merge master, then the tests should be closer to passing now.

**Comment by Magomedov-Magomed** [comment]
> Alright, if you merge master, then the tests should be closer to passing now.

the tests crashed in the blog project, so I changed the code in the blog project, but I couldn't rerun the tests

**Comment by Dreamsorcerer** [comment]
Tests don't seem to be hitting the redis errors anymore. Will continue fixing the other errors in a separate PR.

**Pull request merged**